src/lib/MuteButton.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+    import FaVolumeMute from 'svelte-icons/fa/FaVolumeMute.svelte';
+    import FaVolumeUp from 'svelte-icons/fa/FaVolumeUp.svelte';
+    export let audioRefs: HTMLAudioElement[] = [];
+
+    let soundOn = true;
+
+    setVolume(Number(localStorage.getItem('sound')));
+
+    function toggle() {
+        let newVolume;
+
+        if (localStorage.getItem('sound') == '0') {
+            newVolume = 1;
+            soundOn = true;
+        } else {
+            newVolume = 0;
+            soundOn = false;
+        }
+
+        setVolume(newVolume);
+    }
+
+    function setVolume(volume: number) {
+        localStorage.setItem('sound', String(volume));
+
+        for (let index = 0; index < audioRefs.length; index++) {
+            const player = audioRefs[index];
+            if (player) {
+                player.volume = volume;
+            }
+        }
```

Comment:
for (const audio of audioRefs) {
            audio.muted = Boolean(volume);
        }

Reply:
vorhert mit ! invertieren würde ich sagen weil es ja muted ist und nicht soundOn

Reply:
richtig, hab ich übersehen